taxonomies/food/categories.txt

```diff
@@ -87064,6 +87071,18 @@ en: Liverwurst sausages
 < en:Meat spreads
 en: Liverwurst spreads
 
+< en:Prepared meats
+xx: Teewurst
```

Comment:
For categories, we put en: as the canonical name first, so it should just be:

en: Teewurst
de: Teewurst
xx: Teewurst

Reply:
I changed it!